internal/handlers/handlers_suite_test.go

```diff
@@ -31,6 +33,7 @@ func createTestISO() string {
 	Expect(os.WriteFile(filepath.Join(filesDir, "images/pxeboot/rootfs.img"), []byte("this is rootfs"), 0600)).To(Succeed())
 	Expect(os.WriteFile(filepath.Join(filesDir, "images/pxeboot/vmlinuz"), []byte("this is kernel"), 0600)).To(Succeed())
 	Expect(os.WriteFile(filepath.Join(filesDir, "images/pxeboot/initrd.img"), []byte("this is initrd"), 0600)).To(Succeed())
+	Expect(os.WriteFile(filepath.Join(filesDir, "images/initrd.addrsize"), initrdAddrsize, 0600)).To(Succeed())
```

Comment:
The variable is unnecessary

```suggestion
	Expect(os.WriteFile(filepath.Join(filesDir, "images/initrd.addrsize"), []byte{1, 2, 3, 4, 5, 6, 7, 8, 1, 2, 3, 4, 5, 6, 7, 8}, 0600)).To(Succeed())
```

Reply:
Done